e2e/viewHelper.js

```diff
@@ -117,6 +117,8 @@ export const importWalletWithRecoveryPhrase = async ({
   await Assertions.checkIfVisible(OnboardingCarouselView.container);
   await OnboardingCarouselView.tapOnGetStartedButton();
   await acceptTermOfUse();
+
+  await TestHelpers.delay(3500);
```

Comment:
For me it keeps being not common we need to add this big delay, I can see that after accepting the terms of use we are just navigating to another screen to press a button, I don't understand how that need to have a delay

Can we try instead before accept terms of use , use `device.disableSynchronization()` and after
`device.enableSyncronization`? 
```suggestion
await device.disableSynchronization();
  await acceptTermOfUse();

  await device.enableSynchronization();
```

Reply:
+1 to @tommasini recommendation here. Is it a case of the terms of use bottom sheet not appearing in time?